src/components/Events.tsx

```diff
@@ -0,0 +1,48 @@
+import type { FC } from "react";
```

Comment:
I wonder if the `type` is needed. Sometimes vscode is trying to be smart and adds stuff like this. Restarting it helps. 

I will check it out. No worries :)

Reply:
Was not required after changing type to `React.FC `. 😇